src/main/java/com/google/ehub/utility/BlobstoreURLUtility.java

```diff
@@ -21,24 +21,38 @@
 public final class BlobstoreURLUtility {
   /**
    * Finds the URL that points to the uploaded file in Blobstore.
+   * This method is meant to be used for forms that accept input for only a single file.
    *
    * @param request the request sent by the upload form submission
    * @param formUploadElem the ID of the element in the form that selects the file to upload
-   * @return the upload URL of the submitted file wrapped in an {@link Optional}
+   * @return the upload URL of the submitted file wrapped in an {@link Optional}, the Optional will
+   *     be empty if the form element given has no input that accepts an image file or the URL given
+   *     in the request is malformed
    */
   public static Optional<String> getUploadURL(HttpServletRequest request, String formUploadElem) {
     BlobstoreService blobstoreService = BlobstoreServiceFactory.getBlobstoreService();
 
+    /*
```

Comment:
java style nit: comments inside a function should all be "//"

Reply:
OK I changed the comments to use "//" in last commit.